android-whitelist.json

```
@@ -1120,7 +1120,7 @@
     },
     {
       "group": "com\\.mercadolibre\\.android\\.loyalty_ui_components",
-      "version": "1\\.+|2\\.+"
+      "version": "1\\.+|2\\.+|3\\.+"
```

Tres versiones es un scope muy amplio. ¿Podemos eliminar la `1.+`?
```suggestion
      "version": "2\\.+|3\\.+"
```

---

Listo! gracias por la sugerencia :)